src/services/dashboards/config.ts

```diff
@@ -3,6 +3,15 @@ export const refreshInterval = ['off', '15s', '30s', '1m', '5m', '10m', '30m', '
 export type RefreshInterval = typeof refreshInterval[number];
 
 
+export const DASHBOARD_SCOPE = {
+    DOMAIN: 'DOMAIN',
+    PROJECT: 'PROJECT',
+};
+export const DASHBOARD_VIEWER = {
+    PUBLIC: 'PUBLIC',
+    PRIVATE: 'PRIVATE',
+};
+
```

Comment:
This types exist in `type.ts` of dashboard store. So, I think we need to delete one. Which one is better?